packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -504,6 +518,19 @@ module.exports = React.createClass({
            fixtureName === this.props.fixture;
   },
 
+  userHasComponents() {
+    return Object.keys(this.props.fixtures).length > 0;
+  },
+
+  userHasFixtures() {
+    return _.reduce(this.props.fixtures, (acc, compFixtures) => {
+      const fixtureNames = Object.keys(compFixtures);
+      const noAutoFixture = fixtureNames[0].indexOf('(auto)') === -1;
+
+      return acc || (fixtureNames.length > 1 && noAutoFixture);
```

Comment:
This condition is weird. There can't be an auto fixture when length > 1. How about this?

```js
const hasUserFixtures = fixtureNames.filter(fixtureName => fixtureName.indexOf('(auto)') == -1);
return acc || hasUserFixtures;
```

Actually, why check all components when you can stop after finding first user fixture? E.g.

```js
// good ol for
const { fixtures } = this.props;
for (let component of Object.keys(fixtures)) {
  const fixtureNames = Object.keys(fixtures[component]);
  if (fixtureNames.filter(f => f.indexOf('(auto)') === -1).length > 0) {
    return true;
  }
}

return false;
```

Bonus: Imagine how clean everything would look like if you did this at the beginning of the file:

```js
const { keys } = Object;
```

😙